app/realtime/card_realtime.rb

```diff
@@ -15,7 +15,7 @@ def self.card_created(card, html)
     )
   end
 
-  def self.card_archived(card)
+  def self.card_archived(card, originated_by)
```

Comment:
Unused method argument - `originated_by`. If it's necessary, use `_` or `_originated_by` as an argument name to indicate that it won't be used.